src/main/java/io/debezium/connector/yugabytedb/YugabyteDBSnapshotChangeEventSource.java

```diff
@@ -336,12 +336,17 @@ protected SnapshotResult<YugabyteDBOffsetContext> doExecute(ChangeEventSourceCon
                 final Metronome pollIntervalMetronome = Metronome.parker(Duration.ofMillis(connectorConfig.cdcPollIntervalms()), Clock.SYSTEM);
                 pollIntervalMetronome.pause();
 
+                // This set will contain the tablets for which the server has sent the snapshot
+                // end marker, but we have not received the callback from Kafka - this will ensure
+                // that we do not end up sending redundant GetChanges calls.
+                Set<String> tabletsWaitingForCallback = new HashSet<>();
+
                 String tableId = tableIdToTabletId.getKey();
                 YBTable table = tableIdToTable.get(tableId);
 
                 String tabletId = tableIdToTabletId.getValue();
                 YBPartition part = new YBPartition(tabletId);
-                
+
```

Comment:
These blank lines are the result of IntelliJ auto-formatter and it's removing extra blank spaces automatically, somehow I am unable to get them removed, please ignore them.